app/i18n/de/index.php

```diff
@@ -76,24 +76,24 @@
 		'rss_view' => 'RSS-Feed',
 		'search_short' => 'Suchen',
 		'sort' => array(
-			'_' => 'Sorting criteria',	// TODO
+			'_' => 'Sortierkriterien',
 			'c' => array(
-				'name_asc' => 'Category, feed titles A→Z',	// TODO
-				'name_desc' => 'Category, feed titles Z→A',	// TODO
+				'name_asc' => 'Kategorie, Feed-Titel A→Z',
+				'name_desc' => 'Kategorie, Feed-Titel Z→A',
 			),
-			'date_asc' => 'Publication date 1→9',	// TODO
-			'date_desc' => 'Publication date 9→1',	// TODO
+			'date_asc' => 'Veröffentlichungsdatum 1→9',
+			'date_desc' => 'Veröffentlichungsdatum 9→1',
 			'f' => array(
-				'name_asc' => 'Feed title A→Z',	// TODO
-				'name_desc' => 'Feed title Z→A',	// TODO
+				'name_asc' => 'Feed-Titel A→Z',
+				'name_desc' => 'Feed-Titel Z→A',
 			),
-			'id_asc' => 'Freshly received last',	// TODO
-			'id_desc' => 'Freshly received first',	// TODO
+			'id_asc' => 'Zuletzt empfangen',
+			'id_desc' => 'Zuerst empfangen',
 			'link_asc' => 'Link A→Z',	// TODO
 			'link_desc' => 'Link Z→A',	// TODO
```

Comment:
```suggestion
			'link_asc' => 'Link A→Z',	// IGNORE
			'link_desc' => 'Link Z→A',	// IGNORE
```

If it is the same in German than in English, the lines should be marked as ignored

Reply:
Yeah Link is also Link in German 👍